tests/test_grid3d/test_grid_property.py

```diff
@@ -335,7 +335,7 @@ def test_eclinit_simple_importexport(tmp_path, testdata_path):
     )
     with open(tmp_path / "simple_disc.grdecl") as f:
         fields = f.read().split()
-        assert len(fields) == 17
+        assert len(fields) == 14
         assert fields[3] == "1"  # undef cell for discrete
 
 
```

Comment:
Perhaps we could have a few more tests to see if this is robust?

Reply:
Agree. I have created a new test function that check if file stored with RLE can be read back with the same content